### ISSUE TITLE
Ubuntu Touch won't boot from MultiROM

### ISSUE THREAD

**Issue by ansabhailte** [ubports/ubuntu-touch-for-oneplus-one#39]
I have a OnePlus One, running Exodus 5.1, kernel is AK.291.OPO.CM12.1, MultiROM v33b, TWRP 2016-04-17. I have added Ubuntu Touch rc-proposed 256. I have also tried this with CM12, TWRP 2.8.6.0, and MRM v32j. What always happens is when I select Ubuntu Touch from the boot manager, the screen turns white for a second, and then vibrates and reboots twice back to the boot manager.

**Comment by anidotnet** [comment]
I have COS 13 installed as primary ROM. I have latest multirom manager from play store, and have TWRP-multirom-3.0.2-0_2016-04-17 p1 installed as recovery. I also tried with both Ubuntu Touch rc-proposed 256 and stabe 12, but it always get stuck at boot logo when trying to boot Ubuntu Touch. After few minutes, it reboots automatically to boot selection menu.

**Comment by emmesolutions** [comment]
I have CyanogemMod 12.1 as primary Rom.
Kernel AK.310.OP1.CM
MultiROM v33b
TWRP 2016-04-17
I select Ubuntu Touch from the boot manager, the screen turns white for a second, and then vibrates and reboots twice back to the boot manager.

**Comment by aorzh** [comment]
Same issue. Latest CM 13, TWRP and multirom fom store.

**Comment by foxxtrott141** [comment]
Same issue as well. Hopefully it gets fixed soon though. Here's hoping :/

**Comment by foxxtrott141** [comment]
Running cm14.1 btw

**Comment by Turakar** [comment]
Same problem on my phone.
[Here](https://forums.oneplus.net/threads/wip-ubuntu-touch-for-oneplus-one.266170/page-108#post-15069688) they say that they fixed this problem using an older version of the recovery, however, this did not work for me. However, the download changed the log of the install process on the recovery side: Before I got system-image-upgrade failed, and afterwards the installation works, but the system is still not bootable.
Primary: CM 12.1 and 13.0 tested
Kernel flashed using the Manager App
Tried Ubuntu stable 15, Ubuntu devel proposed 307
MultiROM v33b
TWRP 2016-02-21

**Comment by altagir** [comment]
can we get some feedback ?  does anyone is able to install ubuntu now? 
this is an extremely high level bug... would be nice to have someone investigate

**Comment by UniversalSuperBox** [comment]
We need to get CM12.1-based ports of Ubuntu Touch stable before we can even think about MultiROM, and the stable port is where all of our effort is going right now.

Also, all bugs should now be reported on Launchpad. See https://github.com/ubports/ubuntu-touch-for-oneplus-one/issues/36

Edit: ambiguous statement
